poetry/utils/env.py

```diff
@@ -679,19 +681,38 @@ def create_venv(
 
     @classmethod
     def build_venv(
-        cls, path, executable=None
-    ):  # type: (Union[Path,str], Optional[Union[str, Path]]) -> virtualenv.run.session.Session
+        cls,
+        path,
+        executable=None,
+        with_pip=False,
```

Comment:
I think `create_venv` should now explicitly call `build_venv` with `with_wheel=False, with_setuptools=False` since those are no longer needed in the venv and would be uninstalled?

Reply:
Probably a good thing; but doing so will break so many of the tests as they are currently fragile in that we test called with. We can do this in a later change I think.